docs/integrate/atlan/index.md

```diff
@@ -0,0 +1,78 @@
+# Atlan
+
+```{div} .float-right
+[![Atlan logo](https://website-assets.atlan.com/img/atlan-blue.svg){height=60px loading=lazy}][Atlan]
+```
+```{div} .clearfix
+```
+
+[Atlan] is an active metadata platform for modern data teams, that helps
+them discover, understand, trust, and collaborate on data assets.
+Atlan Pte. Ltd. is a vendor and platform provider for data governance and
+metadata management software. Their product slogan is "Atlan -- The Active
+Metadata Platform".
+
+:::{rubric} Overview
+:::
+
+The Atlan platform serves as a collaborative workspace for data teams,
+allowing organizations to catalog, discover, and govern data assets across
+cloud data warehouses, business intelligence tools, and data pipelines.
+
+Atlan positions itself as addressing what it describes as enterprise "data
+chaos", the challenge organizations face managing data across dozens of tools
+and systems. The platform serves as what the company calls a "metadata control
+plane" that integrates with modern data infrastructure including cloud data
+warehouses like Snowflake and Databricks, as well as business intelligence
+and data science tools.
+
+:::{rubric} Architecture
+:::
+
+Atlan distinguishes itself from traditional data catalogs through its "active
+metadata" architecture, which programmatically harvests metadata such as logs,
+query history, and usage metrics to automate governance tasks. This contrasts
+with passive catalogs that rely on manual metadata entry.
```

Comment:
I learned Alation and Collibra are in a similar area, which may have started with passive catalogs in their infancy.

- https://www.alation.com/product/agentic-data-intelligence-platform/
- https://www.collibra.com/products/collibra-platform
- https://atlan.com/p/alation-vs-atlan/
- https://atlan.com/p/collibra-vs-atlan/
- https://atlan.com/alation-alternatives/
- https://atlan.com/collibra-alternatives/